examples/chip-tool/commands/clusters/WriteAttributeCommand.h

```diff
@@ -105,6 +110,38 @@ class WriteAttribute : public ModelCommand, public chip::app::WriteClient::Callb
         return mWriteClient->SendWriteRequest(device->GetSecureSession().Value());
     }
 
+    template <class T>
+    CHIP_ERROR SendGroupCommand(chip::GroupId groupId, chip::FabricIndex fabricIndex, chip::NodeId senderNodeId,
```

Comment:
This is about sending a group write, not a group command right? If so, we should be clear about the naming, since 'command' really means something different from 'write'.

Reply:
Yes, unfortunately, I don't think that changing the name in this PR is appropriate. The renaming is confusing, yes but it should be changed in another PR as a refactor (a lot stuff will need this refactoring). Moreover this is an override of the mother class `ModelCommands` which doesn't have knowledge of all the possible command types. I'm pretty sure that renaming this will cause a lot of issues and should be done outside of this PR.